### PR TITLE
Fix Pybids depreciation notice

### PR DESCRIPTION
**Pull request by laemtl**

### CODE REVIEW

File: python/lib/bidsreader.py [hunk 1]
```diff
@@ -10,15 +10,24 @@
 
 import lib.exitcode
 import lib.utilities as utilities
+import bids
+
 try:
-    from bids.layout import BIDSLayout
+    from bids import BIDSLayout
```
Comment (cmadjar) [inline python/lib/bidsreader.py:16]:
```suggestion
try:
    from bids import BIDSLayout
except ImportError:
    print("Could not import BIDSLayout. Ensure you have PyBIDS >= 0.12.1 installed.")        
    exit(lib.exitcode.INVALID_IMPORT)
```

If we plan on supporting only PyBIDS > 0.12.1, we should add an error message to tell users to update their pybids package.

**Review by cmadjar — CHANGES_REQUESTED** [review]
One comment on the new imports.

File: python/lib/bidsreader.py (hunk 1 above)
Comment (laemtl) [reply python/lib/bidsreader.py:16]: The change is not supposed to break Pybids < 0.12.1 since I check the version before applying the changes.

**Review by cmadjar — APPROVED** [review]
Tested the changes with pybids-0.11.1 and pybids-0.13 and everything worked like a charm 👍 . Thank you @laemtl :-)

**Pull request merged**